index.html

```diff
@@ -0,0 +1,467 @@
+        <li><a href="#section-5">Variables</a></li>
+        <li><a href="#section-6">Declaring Variables</a></li>
+        <li><a href="#section-7">Variable Scope</a></li>
+        <li><a href="#section-8">Global Variable</a></li>
+        <li><a href="#section-9">constants</a></li>
+        <li><a href="#section-10">Data types</a></li>
+        <li><a href="#section-11">If... Else statement</a></li>
+        <li><a href="#section-12">While statement</a></li>
+        <li><a href="#section-13">Functions Declarations</a></li>
+        <li><a href="#section-14">Reference</a></li>
+      </ul>
+    </div>
+    <!-- Page Content -->
+
+    <div class="main-section">
+      <div class="section" id="section-1">
+        <h1>Introduction</h1>
+        <p>
+          JavaScript is a cross-platform, object-oriented scripting language. It
+          is a small and lightweight language. Inside a host environment (for
+          example, a web browser), JavaScript can be connected to the objects of
+          its environment to provide programmatic control over them.
+        </p>
+        <p>
+          JavaScript contains a standard library of objects, such as Array,
+          Date, and Math, and a core set of language elements such as operators,
+          control structures, and statements. Core JavaScript can be extended
+          for a variety of purposes by supplementing it with additional objects;
+          for example:
+        </p>
+        <ul>
+          <li>
+            Client-side JavaScript extends the core language by supplying
+            objects to control a browser and its Document Object Model (DOM).
+            For example, client-side extensions allow an application to place
+            elements on an HTML form and respond to user events such as mouse
+            clicks, form input, and page navigation.
+          </li>
+          <li>
+            Server-side JavaScript extends the core language by supplying
+            objects relevant to running JavaScript on a server. For example,
+            server-side extensions allow an application to communicate with a
+            database, provide continuity of information from one invocation to
+            another of the application, or perform file manipulations on a
+            server
+          </li>
+        </ul>
+      </div>
+      <div class="section" id="section-2">
+        <h1>What you should already know</h1>
+
+        <p>This guide assumes you have the following basic background:</p>
+
+        <ul>
+          <li>
+            A general understanding of the Internet and the World Wide Web
+            (WWW).
+          </li>
+          <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+          <li>
+            Some programming experience. If you are new to programming, try one
+            of the tutorials linked on the main page about JavaScript.
+          </li>
+        </ul>
+      </div>
```

Comment:
The same comment as above applies here too. You can replace all divs with the section tag with a class of the section they represent. I would suggest you do this for all sections.